vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -30,11 +26,29 @@ There are numerous assets installed as part of the Standard Banking Demo. Entand
 
 ### Prerequisites
 
-- An Entando Application on any Kubernetes provider. Follow one of the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
+- A working instance of Entando running on Kubernetes. [Install Entando on any Kubernetes provider](../#operations) or see [Getting Started](../../docs/getting-started/) for more information. 
 - The ent command line tool, installed and connected to your Kubernetes instance.
 
-### Installation Steps
-
+### Auto Install via the Entando Hub
```

Comment:
In other places we called it Automatic Install (e.g. https://developer.entando.com/v6.3.2/docs/getting-started/#learn-about-entando) I'm not sure shortening it to Auto is clearer.

Reply:
changed it to automatic. I was waffling about it too.